src/js/tech/tech.js

```diff
@@ -805,6 +806,12 @@ Tech.withSourceHandlers = function(_Tech){
     this.disposeSourceHandler();
     this.off('dispose', this.disposeSourceHandler);
 
+    // if we have a source and get another one
```

Comment:
This code was pulled out, as we did not have a valid use case for it. I re-wrote history to do it so this comment still shows up.